app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="layout_linear_margin">5dp</dimen>
+    <dimen name="button_margin_height">60dp</dimen>
+    <dimen name="button_margin_width">90dp</dimen>
+    <dimen name="button_number_margin_height">50dp</dimen>
+    <dimen name="button_number_margin_width">75dp</dimen>
+    <dimen name="txt_result_size">50dp</dimen>
```

Comment:
textSize should be in sp instead of dp

https://developer.android.com/guide/topics/resources/more-resources.html#Dimension